Some changes in docs (#72)
LucidtechAI/cradl-docs: docs/overview/index.md

# What is Cradl?

Cradl is a platform that enables developers to build their own document data capture APIs with deep learning.

Our approach to data capture is quite different than most, and is centered around a single design goal:

:::info Design Philosophy
Be the easiest way for developers to build _customized data capture APIs_ powered by _deep learning models_.

> :::

Reviewer: I think it's weird that the continuation sentence is placed in an info card here. Imo, Info cards are additional / supplementary information and not meant to be a continuation of previous paragraph, ie. it should be possible to have a coherent reading experience without needing to read all the info cards. I suggest completing the sentence outside the info card, but keep a condensed version as the info card.

Reviewer 2: I dont' have a strong opinion on this, so I'm perfectly fine with something like what you describe here.

## Who is Cradl for?

> Cradl is built for developers, and is used by developers e.g. in _financial institutions_, _public sector_, large newspapers and in ERP software companies. If your company:
> 
> * process high volumes of semi-structured documents like invoices, receipts, forms, etc.
> * require high-accuracy data capture
> 
> then Cradl is for you.

Reviewer: Not too fan of the order in which information is presented here. Suggestion:

```suggestion
Cradl is built for developers and works best if:
* You are processing high volumes of semi-structured documents like invoices, receipts, forms, etc.
* You require high-accuracy data capture
```

Reviewer 2: I agree, but: can we find a way to quickly communicate - in the docs - that Cradl is used by serious organizations? Maybe we can have a figure where we put some logos in different quadrants or something? E.g. public sector, financial industry, ERP or something like that?

:::tip Need inspiration?

> Check out some of our [use cases](/use-cases)!

Reviewer: ```suggestion
Check out some of our [use cases](/use-cases)! If you are a developer in the _public sector_, _financial institutions_ or an ERP software company, Cradl might be perfect for you.
```

Reviewer 2: This feels a bit out-of-place to me. I suggest leaving it as is.

:::

## Key concepts
Cradl is centered around the following key concepts which you will frequently encounter in these docs:

> 1. [**Models**](/concepts/models). A model is a _neural network_ that can be trained to solve a specific task, like capturing payment info from invoices.

Reviewer: _Artificial_ neural network? I suggest using machine learning model instead. I think it's a more _colloquial_ / trivial term than neural networks. What do you think?

```suggestion
1. [**Models**](/concepts/models). A model is a _machine learning model_ that can be trained to solve a specific task, like capturing payment info from invoices.
```

Reviewer 2: I kinda agree, but at the same time it feels like a recursvie definition. If we can find an easier/more intuitive definition, that would be preferable IMO, but until we should go with your suggestion.

Reviewer: I agree that definitions such as `model is a model..` is not very informative. However, I would like to argue that in this case it could help the user understand why it is called a `model`. "It's a machine learning model, that's why it's called a model - it makes sense". There are probably other ways of explaining what a model is and get the same _epiphany_ though.

Reviewer 2: I agree.

```suggestion
1. [**Models**](/concepts/models). A model is a _machine learning model_ that can be trained to solve a specific document data extraction task, like capturing payment info from invoices.
```

> 2. [**Documents**](/concepts/documents). A semi-structured document in a PDF or image format. Any language and any layout.

Reviewer: Is the term _semi-structured document_ understood here? Can we just say they are PDF/Images and then give examples instead? Receipts, invoices, forms etc.

Reviewer 2: Alternatively, we can perhaps write something like:
```suggestion
2. [**Documents**](/concepts/documents). A semi-structured document like an invoice, receipt, ID document, etc. in a PDF or image format. Any language and any layout.
```

3. [**Datasets**](/concepts/datasets). A dataset consists of many (typically thousands) of documents that are used to train a specific model. 

> 4. [**Data bundles**](/concepts/training-data). A data bundle is a collection of one or more datasets and is bound to a specific model.

Reviewer: ```suggestion
4. [**Data bundles**](/concepts/training-data). A data bundle is a collection of one or more datasets that has been prepared for **Model** training.
```

> 5. [**Predictions**](/concepts/predictions). When your model is trained on a dataset, it can start capturing data from new and previously unseen documents.

Reviewer: ```suggestion
5. [**Predictions**](/concepts/predictions). When your model has finished training using your **Data Bundle** it can start capturing data from new and previously unseen documents.
```
